pkg/controller/service/BUILD

```diff
@@ -3,9 +3,9 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 go_library(
     name = "go_default_library",
     srcs = [
+        "controller.go",
         "doc.go",
         "patch.go",
-        "controller.go",
```

Comment:
minor comment -- can you squash this commit with the previous one since this is technically part of the file rename?

Reply:
incase it's not clear via github UI, squash last commit with 2nd last commit

Reply:
commits squashed